packages/holocron/__tests__/loadModule.node.spec.js

```diff
@@ -62,13 +57,26 @@ describe('loadModule.node', () => {
     return require('../src/loadModule.node.js').default; // eslint-disable-line global-require
   }
 
+  function getSHA(content = 'the Spanish Inquisition') {
+    // eslint-disable-next-line global-require
+    return require('ssri').fromData(content, { algorithms: ['sha256', 'sha384'] }).toString();
+  }
```

Comment:
This function is needed to get the mocked content and extract a valid `sha` for testing production only conditions.